### PR TITLE
Make "back" buttons nicer

### LINKED ISSUE #64: Improve "change xxxx" options in last step
They currently look like this:

image

I think it's not very readable and we could probably come up with something better. One option would be two buttons, one with a one-arrow icon and the other one with a "fast" forward double-arrow icon.
I also think that making step indicators clickable would help a great deal with UX.

image

### PR DESCRIPTION
**Pull request by pferreir**
The "back" buttons now look nicer:
image

The `Navigator` was also improved. The steps are now clickable:
image

Closes #64

### CODE REVIEW

File: newdle/client/src/components/Navigator.js [hunk 1]
```diff
@@ -21,34 +22,54 @@ export default function Navigator() {
...
-          {index + 1}
-        </Step>
-      ))}
+      <ul className={styles['step-box']}>
+        {steps.map((_, index) => (
+          <Step
+            key={index}
+            index={index + 1}
+            active={index + 1 === active}
+            onClick={index + 1 < active ? () => dispatch(setStep(index + 1)) : null}
```
Comment (ThiefMaster) [inline newdle/client/src/components/Navigator.js]: I'd allow navigating forward as well, and only disallow going to step 3 if there are no timeslots defined (could add a condition to the corresponding `steps` object that runs the selector, like we do to disable the next button on the step's page)

Comment (pferreir) [reply newdle/client/src/components/Navigator.js]: Done.

File: newdle/client/src/components/Navigator.js [hunk 2]
```diff
@@ -21,34 +23,56 @@ export default function Navigator() {
...
-        </Step>
-      ))}
+      <ul className={styles['step-box']}>
+        {steps.map((__, index) => (
+          <Step
+            key={index}
+            index={index + 1}
+            active={index + 1 === active}
+            // Step 3 (index === 2) should only be clickable if there are already slots defined
+            onClick={index !== 2 || !_.isEmpty(slots) ? () => dispatch(setStep(index + 1)) : null}
```
Comment (ThiefMaster) [inline newdle/client/src/components/Navigator.js]: Passing `null` doesn't use the default prop, only `undefined` does.

Comment (pferreir) [reply newdle/client/src/components/Navigator.js]: Does it really matter, though? The child component doesn't misbehave when `onClick=null`.

Comment (pferreir) [reply newdle/client/src/components/Navigator.js]: But OK, it's prettier, since otherwise the default value would never be used...

Comment (ThiefMaster) [reply newdle/client/src/components/Navigator.js]: no, i just noticed that you are specifying a default that's never used

Comment (pferreir) [reply newdle/client/src/components/Navigator.js]: Yeah, I'm fixing it, thanks 😉

**Pull request merged**